README.md

```diff
@@ -31,24 +31,34 @@ pyproject.toml vs setup.py
    To coincide with these changes, some changes to tox.ini and the addition of a MANIFEST.ini file were also necessary.
 
```

Comment:
The verbiage for the `pyproject.toml vs setup.py` needs cleaned up. It still instructions about the `requirements.in` file which is not being used any longer.